datadog/resource_datadog_logs_archive_test.go

```diff
@@ -0,0 +1,330 @@
+package datadog
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/logging"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"net/http"
+	"os"
+)
+
+//Test
+// create: OK azure
+
+const archiveAzureConfigForCreation = `
+resource "datadog_logs_archive" "my_azure_archive" {
+	name = "my first azure archive"
+	query = "service:toto"
+	azure = {
+		container 		= "my-container"
+		tenant_id 		= "my-tenant-id"
+		client_id       = "testc7f6-1234-5678-9101-3fcbf464test"
+		storage_account = "storageAccount"
+		path            = "/path/blou"
+	}
+}
+`
+
+func TestAccDatadogLogsArchiveAzure_basic(t *testing.T) {
+	rec := initRecorder(t)
+	defer rec.Stop()
+	httpClient := &http.Client{Transport: logging.NewTransport("Datadog", rec)}
+	// At the moment there's no azure integration in tf so we manually:
```

Comment:
Should we wait for https://github.com/terraform-providers/terraform-provider-datadog/pull/556 ?